packages/@react-aria/tag/src/useTagGroup.ts

```diff
@@ -11,17 +11,23 @@
  */
 
 import {AriaLabelingProps, DOMAttributes, DOMProps} from '@react-types/shared';
-import {filterDOMProps, mergeProps} from '@react-aria/utils';
+import {filterDOMProps, mergeProps, useId} from '@react-aria/utils';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
 import {RefObject, useState} from 'react';
 import {TagGroupProps} from '@react-types/tag';
 import type {TagGroupState} from '@react-stately/tag';
 import {TagKeyboardDelegate} from './TagKeyboardDelegate';
 import {useFocusWithin} from '@react-aria/interactions';
 import {useGridList} from '@react-aria/gridlist';
 import {useLocale} from '@react-aria/i18n';
+import {useLocalizedStringFormatter} from '@react-aria/i18n';
 
 export interface TagGroupAria {
-  tagGroupProps: DOMAttributes
+  /** Props for the tag grouping element. */
+  tagGroupProps: DOMAttributes,
+  /** Props for an optional group of actions. */
+  tagGroupActionsProps: DOMAttributes
```

Comment:
Should this be singular or plural? Right not there is only one action.

Reply:
The group could contain multiple actions, i.e. the custom action and the 'show all' button. Plus at the react-aria level, people may want more.